editor/plugins/abstract_polygon_2d_editor.cpp

```diff
@@ -525,7 +529,7 @@ void AbstractPolygon2DEditor::forward_canvas_draw_over_viewport(Control *p_overl
 		}
 
 		if (!wip_active && j == edited_point.polygon && EDITOR_GET("editors/polygon_editor/show_previous_outline")) {
-			const Color col = Color(0.5, 0.5, 0.5); // FIXME polygon->get_outline_color();
```

Comment:
I looked into it with gitblame. This FIXME is from a 6 years old PR when our process wasn't as stringent. The reason for it appears to be lost in time. The only outline_color in our codebase is for GUI text.